status/status.go

```diff
@@ -61,11 +61,7 @@ func (a *Adapter) GetReporters(ctx context.Context, pipelineRun *tektonv1beta1.P
 	var reporters []Reporter
 
 	if helpers.HasLabelWithValue(pipelineRun, gitops.PipelineAsCodeGitProviderLabel, gitops.PipelineAsCodeGitHubProviderType) {
-		reporter, err := a.githubReporter(ctx, a.logger, a.k8sClient, pipelineRun)
```

Comment:
Looks like `ctx` parameter won't be needed anymore for the `GetReporters` function.

Reply:
Removed.

Reply:
Thanks!